rust/Cargo.toml

```diff
@@ -65,15 +65,24 @@ accelerate-src = "0.3.2"
 [target.'cfg(target_os = "linux")'.dependencies]
 openblas-src = { version = "0.10.8", default-features= false, features = ["static", "cblas"]}
 
+[target.'cfg(target_os = "windows")'.dependencies]
```

Comment:
Should we update README to reflect the change on windows?

Reply:
Let's say if ppl are using this on windows, do they need to install blas and etc. ?